applications/index.html

```diff
@@ -77,8 +77,14 @@ <h2>Status</h2>
         </button>
         <div class="application-details-main"></div>
         <div class="application-details-actions">
-          <button class="application-details-reject">Reject</button>
           <button class="application-details-accept">Accept</button>
+          <p class="application-details-reject">Reject</p>
+          <p class="application-details-accepted-msg">
+            Application was already accepted
+          </p>
+          <p class="application-details-rejected-msg">
+            Application is already rejected
+          </p>
```

Comment:
- instead of keeping this value always in dom can we conditionally add the message depending on the status?

Reply:
We can. Doing this

Reply:
This is done @yesyash